frontend/components/player/player_container.js

```diff
@@ -0,0 +1,23 @@
+import { connect } from "react-redux";
+import Player from "./player";
+
+import { reduxPlay } from "../../actions/now_playing_actions";
+
+const mapStateToProps = (state, ownProps) => {
+	return {
+		currentUser: state.entities.users[state.session.id],
+		errors: state.entities.errors,
+		params: ownProps.params,
+		path: ownProps.path,
+		history: ownProps.history,
```

Comment:
What's history for again?

Reply:
`History` is a key given in the `match` object which is provided by all Rails Router AuthRoutes (simulates website history without actually changing the page), but because this component is a few levels deep from the original AuthRoute, `params/path/history` had already been deconstructed from `match`.... I had mentioned the `match` object in a comment in the other container, and you had commented that the second comment line was unnecessary? Should I actually add a comment here?

Reply:
In case the component needed access to browser history but you're right, Player doesn't. Thanks.